src/app/pages/posts/posts.component.ts

```diff
@@ -9,39 +9,8 @@ import { SessionService } from '../../services/session.service'
 })
 
 export class PostsComponent implements OnInit {
-post:{
-    id:number,
-    title:string,
-    first_name:string,
-    category_id:number,
-    customer_id:number,
-    post_status_id:number,
-    post_priority: number,
-    photo:string,
-    description:string,
-    city:string,
-    state:string,
-    zip_code:number,
-    budget:number,
-    can_bid:boolean,
-    created_at:string,
-} = {
-    id:null,
-    title:'',
-    first_name:'',
-    category_id:null,
-    customer_id:null,
-    post_status_id:null,
-    post_priority: null,
-    photo:'',
-    description:'',
-    city:'',
-    state:'',
-    zip_code:null,
-    budget:null,
-    can_bid: false,
-    created_at:'',
-}
+post: any;
```

Comment:
Believe the post is supposed to have explicit field definitions rather than `any`

Reply:
Jen's working on setting up the interfaces file right now and this should be resolved today